business/src/main/java/org/mskcc/cbio/portal/service/ApiService.java

```diff
@@ -12,7 +12,12 @@
 import org.cbioportal.model.Mutation;
```

Comment:
I think I was using it to create an endpoint to test if it would query correctly